libs/designsystem/src/lib/components/chart/chart-type.ts

```diff
@@ -4,5 +4,5 @@ export enum ChartType {
   AREASPLINE = 'areaspline',
   TIMESERIES = 'timeseries',
   ACTIVITYGAUGE = 'activitygauge',
-  // SOLIDGAUGE = 'solidgauge',
+  Custom = 'custom',
```

Comment:
When we don't want to have any of the default configurations

Reply:
Consider moving last in list for readability